core/common/src/main/java/alluxio/exception/runtime/AlluxioRuntimeException.java

```diff
@@ -192,6 +192,19 @@ public static AlluxioRuntimeException from(IOException ioe) {
     return new UnknownRuntimeException(ioe);
   }
 
+  /**
+   * Converts an Error to a corresponding runtime exception.
+   *
+   * @param e error
+   * @return the corresponding status exception
+   */
+  public static AlluxioRuntimeException from(Error e) {
+    if (e instanceof OutOfMemoryError) {
```

Comment:
Do we want to recover from OOM error? I mean I know OOM will not directly crash the process but I don't think recovering from OOM is a great idea. Also the OOM here is probably "Out of direct memory" but OOM can also be "failed to create native threads" or "out of heap", we can't probably recover from those. WDYT @dbw9580 ?

Reply:
Java and GC have been the error.

Reply:
I got it. I think there should be a parameter to decide whether to retry.

Reply:
I got it. It was fixed.